specs-go/config.go

```diff
@@ -218,6 +218,8 @@ const (
 	UTSNamespace LinuxNamespaceType = "uts"
 	// UserNamespace for isolating user and group IDs
 	UserNamespace LinuxNamespaceType = "user"
+	// ImaNamespace for isolating PCR values
+	ImaNamespace LinuxNamespaceType = "ima"
```

Comment:
https://github.com/opencontainers/runtime-spec/blob/main/config-linux.md needs to be updated too